Services/DoctorService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using MapsterMapper;
+using Microsoft.EntityFrameworkCore;
+using ProfilesMicroService.Data;
+using ProfilesMicroService.Models;
+using ProfilesMicroService.Models.Dto;
+using ProfilesMicroService.Services.Interfaces;
+using System.Security.Cryptography;
+
+namespace ProfilesMicroService.Services;
+public class DoctorService : IDoctorService
+{
+    private readonly ApplicationDbContext _context;
+    private readonly IMapper _mapper;
+
+    public DoctorService(ApplicationDbContext context, IMapper mapper)
+    {
+        _context = context;
+        _mapper = mapper;
+    }
+
+    public async Task<List<DoctorDto>> GetDoctorsAsync()
```

Comment:
to return data, let's use the collections interface rather than the implementations themselves